sdk/storage/azure-storage-file-share/azure/storage/fileshare/aio/_file_client_async.py

```diff
@@ -210,10 +210,10 @@ def __init__(
 
     @classmethod
     def from_file_url(
-            cls, file_url: str,
-            snapshot: Optional[Union[str, Dict[str, Any]]] = None,
-            credential: Optional[Union[str, Dict[str, str], "AzureNamedKeyCredential", "AzureSasCredential", "AsyncTokenCredential"]] = None,  # pylint: disable=line-too-long
-            **kwargs: Any
```

Comment:
These were not merge glitches-- they in fact are too indented in `main`

Reply:
Nice catch! Looks like this didn't affect other files so we're good to go :P